api/pyproject.toml

```diff
@@ -9,14 +9,11 @@ license = "MIT"
 cachetools = "^5.3.1"
 python = "^3.10.4"
 numpy = "^1.26.2"
-python-jose = {extras = ["cryptography"], version = "^3.3.0"}
-fastapi = "^0.109.2"
-uvicorn = {extras = ["standard"], version = "^0.27.1"}
-gunicorn = "^21.2.0"
```

Comment:
gunicorn is unused and fastapi comes with uvicorn?

Reply:
Thats right!